packages/plugins/plugin-appsflyer/src/AppsflyerPlugin.tsx

```diff
@@ -47,11 +48,15 @@ export class AppsflyerPlugin extends DestinationPlugin {
     }
 
     if (
-      clientConfig?.trackDeepLinks === true &&
-      !this.hasRegisteredDeepLinkCallback
+      (clientConfig?.trackDeepLinks === true &&
+        !this.hasRegisteredDeepLinkCallback) ||
+      !this.hasRegisteredUnifiedLinkCallback
     ) {
       this.registerDeepLinkCallback();
       this.hasRegisteredDeepLinkCallback = true;
+
+      this.registerUnifiedDeepLinkCallback();
+      this.hasRegisteredUnifiedLinkCallback = true;
```

Comment:
We don't need another flag for this one do we? Since they both get added at the same time